cobbler/items/item.py

```diff
@@ -34,6 +96,79 @@ class Item:
     TYPE_NAME = "generic"
     COLLECTION_TYPE = "generic"
 
+    # Item types dependencies.
+    # Used to determine descendants and cache invalidation.
+    # TODO: Use decorators?
+    # TODO: Use to determine the order of deserialization and get the ability
```

Comment:
If we merge this PR with roughly the code we have now, I would wish to see all your ideas put into a feature issue and remove all the TODOs from this PR. I would schedule these for 3.4.x or 3.5.0 already to have a clear estimate. I see that it is much easier for me to track TODOs as GitHub tickets instead of TODOs rotting in the code.

Reply:
fixed in https://github.com/cobbler/cobbler/pull/3266/commits/d8117a8cb4f51494abf7ebc60d0d9421704cf042